### PR TITLE
Add jossemarGT to sig-docs-es-reviews

### PR DESCRIPTION
**Pull request by electrocucaracha**
/assign @ramrodo

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/language es

**Comment by ramrodo** [comment]
Congrats, @jossemarGT!
/lgtm

**Review by krol3 — APPROVED** [review]
Nice job, @jossemarGT 
/lgtm

**Comment by ramrodo** [comment]
/assign @sftim

 We want to add @jossemarGT to sig-docs-es-reviewers. Could you help us with this, please?

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -59,6 +59,7 @@ aliases:
     - electrocucaracha
     - raelga
     - ramrodo
+    - jossemarGT
```
Comment (sftim) [inline OWNERS_ALIASES:62]: Ideally, these would be in alphabetical order; it helps with maintenance.
@electrocucaracha would you be willing to fix that?

Comment (electrocucaracha) [reply OWNERS_ALIASES:62]: Sure no problem, easy to change it

**Review by divya-mohan0209 — COMMENTED** [review]
/approve

**Pull request merged**